seq2science/rules/get_fastq.smk

```diff
@@ -150,10 +150,14 @@ rule sra2fastq_PE:
         --threads {threads} --split-e --skip-technical --dumpbase \
         --readids --clip --read-filter pass --defline-seq '@$ac.$si.$sg/$ri' \
         --defline-qual '+' --gzip >> {log} 2>&1
+        
+        # check if the files exist
+        if ! compgen -G "{output.tmpdir}/*_1*" > /dev/null ; then printf "ERROR: Couldn't find read 1.fastq after dumping! Perhaps this is not a paired-end file?\n" >> {log} 2>&1; fi
+        if ! compgen -G "{output.tmpdir}/*_2*" > /dev/null ; then printf "ERROR: Couldn't find read 2.fastq after dumping! Perhaps this is not a paired-end file?\n" >> {log} 2>&1; fi
```

Comment:
I did not know that echo doesn't work :sweat_smile: ... I did just now realize that the newline isn't even necessary, as echo automatically makes a newline. Do we know if printf is standard in bash?